File: JanperCab/3-Application/Dtos/DuraformPriceGrid/DuraformPriceGridDto.cs
namespace _3_Application.Dtos.DuraformPriceGrid
{
    public abstract class DuraformPriceGridDto
    {
        public int Id { get; set; }

        public decimal MinHeight { get; set; }

        public decimal MaxHeight { get; set; }

        public decimal MinWidth { get; set; }

        public decimal MaxWidth { get; set; }

        public decimal Price { get; set; }

        public decimal TempPrice => Price;

        public int DuraformSerieId { get; set; }
    }
}

File: JanperCab/5-JanperCab/ClientApp/src/app/_models/duraform-price/DuraformPriceGridDto.ts
export abstract class DuraformPriceGridDto {
  $type: string =
    '_3_Application.Dtos.DuraformPriceGrid.DuraformPriceGridDto, 3-Application, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null';
  id: number = 0;
  duraformSerieId: number;
  minHeight: number;
  maxHeight: number;
  minWidth: number;
  maxWidth: number;
  price: number;
  tempPrice: number;
}
